source/production/F0.Minesweeper.Logic/Utilities.cs

```diff
@@ -0,0 +1,42 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using F0.Minesweeper.Logic.Abstractions;
+
+namespace F0.Minesweeper.Logic
+{
+	static class Utilities
+	{
+		internal static IEnumerable<Location> GetLocationsAreaAroundLocation(IEnumerable<Location> allLocations, Location location, bool exludeGivenLocation)
+		{
+			var returnLocations = new List<Location>();
```

Comment:
We could also use a `HashSet` instead of a `List` since we only want to cover unique locations which would get us rid of the `Distinct` operation at the end as well.

Reply:
Awesome idea, Thanks! 😄